WordPress/src/main/java/org/wordpress/android/ui/dataview/DataViewItemCard.kt

```diff
@@ -61,6 +57,11 @@ fun DataViewItemCard(
                 RemoteImage(
                     imageUrl = image.imageUrl,
                     fallbackImageRes = image.fallbackImageRes,
+                    modifier = modifier
```

Comment:
I think the `modifier` from the `DataViewItemCard` arguments should only be passed to the first composable item. So, in this case, we may need to create new one

```suggestion
                    modifier = Modifier
```

Reply:
Thanks for catching that! It was a typo on my part. Resolved in [c91a8bf](https://github.com/wordpress-mobile/WordPress-Android/pull/21969/commits/c91a8bf44162f4546e7ec32888c699bd6066e058).